### PR TITLE
Add Katello 4.13.1 release notes

### PR DESCRIPTION
**Pull request by parthaa**
* [x] I am okay with my commits getting squashed when you merge this PR.
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/katello-4.13.1.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1857[Redmine]
+
+== Katello
+
+=== Container
+
+* pass:[Create Katello push repositories as needed at container push time] - https://projects.theforeman.org/issues/37455[#37455]
+
+=== Content Views
```
Comment (Lennonka) [inline guides/doc-Release_Notes/topics/katello-4.13.1.adoc:11]:
```suggestion
=== Content views
```

File: guides/doc-Release_Notes/topics/katello-4.13.1.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+* pass:[CV promote fails with undefined method `get_status' for nil:NilClass when deleting a Host in the CV during Finalize phase of the Promote task] - https://projects.theforeman.org/issues/37543[#37543]
+
+=== Foreman Proxy Content
```
Comment (Lennonka) [inline guides/doc-Release_Notes/topics/katello-4.13.1.adoc:15]:
```suggestion
=== Foreman Proxy content
```

**Review by Lennonka — CHANGES_REQUESTED** [review]
We use sentence case for headings.
The rest looks good.

**Review by Lennonka — APPROVED** [review]

**Comment by ekohl** [comment]
I've reverted the proposed changes by @Lennonka and will merge it now since it was already released a while back and announced. The procedure should make it explicit: first publish release notes, then announce.

**Pull request merged**